DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/ReviewConfirmationRequests.cshtml

```diff
@@ -0,0 +1,126 @@
+        <thead role="rowgroup" class="nhsuk-table__head">
+          <tr role="row">
+            <th role="columnheader" class="nhsuk-u-width-one-half" scope="col">
+              Competency
+            </th>
+            <th role="columnheader" scope="col">
+              Question
+            </th>
+            <th role="columnheader" scope="col">
+              Response
+            </th>
+            <th role="columnheader" scope="col">
+              Request
+            </th>
+          </tr>
+        </thead>
+        <tbody class="nhsuk-table__body">
+        @foreach (var competency in Model.Competencies)
+        {
+          foreach(var question in competency.AssessmentQuestions)
+          {
+          <tr role="row" class="nhsuk-table__row">
+            <td role="cell" class="nhsuk-table__cell nhsuk-u-font-size-16">
+              <span class="nhsuk-table-responsive__heading">Comptenecy </span>
+              @DisplayStringHelper.RemoveMarkup(competency.Description)
+            </td>
+            <td role="cell" class="nhsuk-table__cell nhsuk-u-font-size-16">
+              <span class="nhsuk-table-responsive__heading">Question </span>
+              @DisplayStringHelper.RemoveMarkup(question.Question)
+            </td>
+            <td role="cell" class="nhsuk-table__cell nhsuk-u-font-size-16">
+              <span class="nhsuk-table-responsive__heading">Response </span>
+              <partial name="../../Supervisor/Shared/_AssessmentQuestionResponse" model="question" />
+            </td>
+            <td role="cell" class="nhsuk-table__cell nhsuk-u-font-size-16">
+              <span class="nhsuk-table-responsive__heading">Request </span>
+              @competency.SupervisorName<br />
+              @competency.SupervisorVerificationRequested?.ToString("dd/MM/yyyy")<br />
+              <span style="white-space: nowrap;">
+                @if (competency.EmailSent.HasValue && DateTime.UtcNow.Subtract(competency.EmailSent.Value) > TimeSpan.FromHours(1))
+                {
+                  <a asp-action="ResendSupervisorVerificationRequest"
+                     asp-route-selfAssessmentId="@Model.SelfAssessment.Id"
+                     asp-route-vocabulary="@Model.SelfAssessment.Vocabulary"
+                     asp-route-candidateAssessmentSupervisorId="@competency.CandidateAssessmentSupervisorId"
+                     asp-route-selfAssessmentResultId="@question.ResultId">Resend</a>
+                }
+                <a asp-action="WithdrawSupervisorVerificationRequest"
+                   asp-route-selfAssessmentId="@Model.SelfAssessment.Id"
+                   asp-route-supervisorVerificationId="@competency.SupervisorVerificationId">Withdraw</a>
+              </span>
+            </td>
+          </tr>
+          }
+        }
+        </tbody>
+      </table>
+    }
+    <div class="nhsuk-grid-row nhsuk-u-margin-top-4">
+      <div class="nhsuk-grid-column-full">
+        <a class="nhsuk-button"

```

Comment:
This button should be outside of the `@if Model.competencies.Any()' conditional statement so that new requests can be submitted if none already exist.